docs/source/examples/Using Interact.ipynb

```diff
@@ -898,6 +898,96 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### `interactive_output`\n",
+    "\n",
+    "The `interactive_output` function provides maximum flexibility at the cost of a bit more code.\n",
+    "\n",
+    "It does not generate a user interface for the widgets, like `interact`, `interactive`, and `interact_manual` do. This is powerful, because it means you can create a widget, put it in box, and then pass the widget to `interactive_output`, and have control over the widget and its layout."
```

Comment:
Maybe `like ... do` -> `unlike` (to make the opposition even clearer).

`put it in box` -> `put it in a box`